packages/api-server/src/index.ts

```diff
@@ -11,9 +11,14 @@ if (process.argv0 === 'api-server') {
   console.log()
 }
 
-const { port, socket } = yargs
+const { port, socket, routePrefix } = yargs
   .option('port', { default: 8911, type: 'number', alias: 'p' })
-  .option('socket', { type: 'string' }).argv
+  .option('socket', { type: 'string' })
+  .option('routePrefix', {
+    default: '/',
+    type: 'string',
+    desc: 'Route prefix where your api functions are served',
+  }).argv
```

Comment:
Maybe instead of the helper function you can coerce the input into something that's usable from the start.

https://yargs.js.org/docs/#coerce